### DISCUSSION TITLE
GLSL 1.20

### DISCUSSION THREAD

**Discussion by tobiasgrossmann** [dosbox-staging/dosbox-staging#4133] [Q&A]
Hi, I run a xubuntu on kernel 6.8.0-51 on a macbook from 2016. 

I get this error and cant find any solution so far: 

```
2024-12-30 18:16:56.538 | OPENGL: Error compiling shader: 0:1(10): error: GLSL 1.20 is not supported. Supported versions are: 1.00 ES, 3.00 ES, 3.10 ES, and 3.20 ES

2024-12-30 18:16:56.538 | OPENGL: Failed to compile shader
```

Yes I want GL as I want shaders. 

OpenGL version string: 4.6 (Compatibility Profile) Mesa 24.0.9-0ubuntu0.3

Thanks. 

Update: Instead of the homebrew version I downloaded the sources of version 0.81, compiled and it works.

**Comment by johnnovak** [comment]
> Update: Instead of the homebrew version I downloaded the sources of version 0.81, compiled and it works.

How about compiling 0.82.0? Can you also try the Linux .tar.gz from our releases page? If all these work, the Homebrew folks screwed something up and you should raise a ticket with them.

To be clear, we don't support anything else than our official releases from our [releases page](https://www.dosbox-staging.org/releases/). For any 3rd party repackagers, you'd need to go and complain to them if they broke something. I strongly advise to only use our .tar.gz on Linux, or alternatively build your own.
